bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -205,18 +206,23 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
                 };
             });
 
+            languageCode = mainLanguageCode;
```

Comment:
I would remove the `mainLanguageCode` variable and set `type.get('mainLanguageCode')` as the value of `languageCode` variable.